harmony/harmony.py

```diff
@@ -455,13 +465,16 @@ def _session(self):
 
     def _submit_url(self, request: Request) -> str:
         """Constructs the URL for the request that is used to submit a new Harmony Job."""
-        variables = [v.replace('/', '%2F') for v in request.variables]
-        vars = ','.join(variables)
-        return (
-            f'{self.config.root_url}'
-            f'/{request.collection.id}'
-            f'/ogc-api-coverages/1.0.0/collections/{vars}/coverage/rangeset'
-        )
+        if request.capabilities:
```

Comment:
Similar comment instead of calling `submit` on the harmony client - maybe add `get_capabilities` and pass in the Capabilities object as a parameter.

Reply:
I think it would be an overkill once we make a different request type for capabilities request. The harmony client would still call `submit` to submit a CapabilitiesRequest and that will return the collection capabilities.

Reply:
The name `submit` might be confusing, but its intention (including all of the documentation) is to submit a transformation request using the OGC coverages API. It's not a generic `submit` as in call any arbitrary HTTP endpoint with a harmony root. I don't think we should overload it and then add if statements to change what API it calls.

Reply:
I'm okay with using `submit` if you are creating an entirely separate `CapabiliteisRequest` class that also has a `submit` method. Not sure of the semantics of what it would return though; since those requests are always synchronous it doesn't make sense to query the progress, etc.

Reply:
The main reason I want to reuse the `submit` function to handle two different requests is to reuse the common functions that are needed to for authentication and parameter handling. Since most of the request submission is the same for both requests, it make sense to share the functions rather than duplicate the functions. Using if statements is the easiest way to achieve this as Python lack the ability to overload functions by parameter type.